src/HookDocsRule.php

```diff
@@ -216,7 +216,7 @@ protected function validateSingleParamTag(string $paramName, ParamTag $paramTag,
             $this->currentScope->isDeclareStrictTypes()
         );
 
-        if (! $accepted) {
+        if ($accepted->result) {
```

Comment:
```suggestion
        if (! $accepted->result) {
```

I think. Oversight from rebasing?